04-GitRStudio.Rmd

```diff
@@ -28,18 +29,18 @@ knitr::include_graphics("images/Git5a.PNG")
 You will notice a few option here when setting up your new repo.
 
 -   Repository name (required): Think short but concise. No spaces permitted. Also, each repo must have a unique name.
+It can be helpful to have your RStudio project folders have the same name as the repository.
 
--   Public or Private repo: if you create a public repo, anyone on Github can see your work and seamless collaborate with you. A private repo can only be seen and accessed by you. You can add individual permissions to a Private repo to allow collaboration. And you can change the privacy setting at a later date.
+-   Public or Private repo: if you create a public repo, anyone on Github can see your work and seamlessly collaborate with you. A private repo can only be seen and accessed by you. You can add individual permissions to a Private repo to allow collaboration. And you can change the privacy setting at a later date.
 
 -   Initialize this repository with: `README`, `.gitignore`, `license`: it is good practice to include all three of these files in your repo.
 
 `README` file can be used to add a short description of your project.
 
-`.gitignore` will have a drop down menu of templates. Select the template specific to R. This files tell .git which files types to ignore in the repo (i.e. they won't be tracked or synced by Git). For example, the session data files `.RData` and history files `.Rhistory` will not be backed up. You may also be working with very large raw data files that are too big for GitHub to handle. These can be added to the `.gitignore` file.
+`.gitignore` will have a drop down menu of templates. Select the template specific to R. This files tell Git which files types to ignore in the repo (i.e. they won't be tracked or synced by Git). For example, the session data files `.RData` and history files `.Rhistory` will not be backed up. You may also be working with very large raw data files that are too big for GitHub to handle. These can be added to the `.gitignore` file.
 
 `Choose a license` will also have a drop down menu. It is wise to protect your intellectual property by assigning a license to your scripts and associated documents. This is not mandatory, but recommend. For most of our work, the GNU General Public License v3.0 is a good choice. You can learn more about your options [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository).
 
-> Recall: all .git repos that are backup on the cloud are initialized from Github. This is referred to the `origin` repo.
```

Comment:
I'm not sure this is strictly correct. You can absolutely initialize a Git repo locally and then link it to a GitHub repo. The origin will be the GitHub one, though, for sure...